fl4health/losses/mkmmd_loss.py

```diff
@@ -167,12 +216,50 @@ def forward(self, Xs: torch.Tensor, Xt: torch.Tensor) -> torch.Tensor:
         #   G is (kernel_num, kernel_num)
         #   h is (kernel_num, 1)
         #   A is (1, kernel_num)
-        #   b is ()
-        beta = self.sign * QPFunction(verbose=False)(Q, p, G, h, A, b)
+        #   b is a scalar
+
+        # QPFunction returns betas in the shape 1 x num_kernels to we take the transpose for consistency
+        return QPFunction(verbose=False, solver=QPSolvers.CVXPY)(regularized_Q_k, p, G, h, hat_d_per_kernel.t(), b).t()
+
+    def optimize_betas(self, X: torch.Tensor, Y: torch.Tensor, lambda_m: float) -> torch.Tensor:
```

Comment:
Maybe have a default value of 1e-5 for  lambda_m?

Reply:
Yup that's good with me.